packages/file_selector/file_selector/README.md

```diff
@@ -92,5 +108,14 @@ pass different `XTypeGroup`s based on `Platform`.
 
 † `mimeTypes` are not supported on version of macOS earlier than 11 (Big Sur).
 
+### Features supported by platform
+
+| Feature                | Description                        | iOS      | Linux      | macOS    | Windows      | Web         |
+| ---------------------- |----------------------------------- |--------- | ---------- | -------- | ------------ | ----------- |
+| Choose a single file   | Pick a file/image                  | ✔️       | ✔️        | ✔️       | ✔️          | ✔️          |
+| Choose multiple files  | Pick multiple files/images         | ✔️       | ✔️        | ✔️       | ✔️          | ✔️          |
+| Save a file            | Pick a directory to save a file in | ❌       | ✔️        | ✔️       | ✔️          | ❌          |
```

Comment:
We should call this "Choose a save location" since it doesn't actually save anything. (The example earlier in the file is titled "Save a file" because it shows the full flow of saving something.)